eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/OperationMethodProviders/ResourceOperationMethodProvider.cs

```diff
@@ -86,27 +85,22 @@ protected virtual MethodSignature CreateSignature()
                 _convenienceMethod.Signature.NonDocumentComment);
         }
 
-        private MethodBodyStatement BuildDiagnosticScopeDeclaration(out VariableExpression scopeVariable)
-        {
-            return UsingDeclare(
-                "scope",
-                typeof(DiagnosticScope),
-                _clientDiagnosticsField.Invoke(
-                    nameof(ClientDiagnostics.CreateScope),
-                    [Literal($"{_resourceClientProvider.Name}.{_signature.Name}")]),
-                out scopeVariable);
-        }
-
         private TryExpression BuildTryExpression()
         {
             var cancellationTokenParameter = _convenienceMethod.Signature.Parameters.Single(p => p.Type.Equals(typeof(CancellationToken)));
 
+            var requestMethod = _serviceMethod.GetCorrespondingRequestMethod(_resourceClientProvider);
```

Comment:
should remove GetCorrespondingRequestMethod and use https://github.com/Azure/azure-sdk-for-net/blob/ceee0af493744f1dbcb40aa0a3274f761f10018f/eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Extensions/ClientProviderExtensions.cs#L12